core/src/test/scala/unit/kafka/admin/AclCommandTest.scala

```diff
@@ -325,11 +349,15 @@ class AclCommandTest extends QuorumTestHarness with Logging {
 
   private def withAuthorizer()(f: Authorizer => Unit): Unit = {
     val kafkaConfig = KafkaConfig.fromProps(brokerProps, doLog = false)
-    val authZ = new AclAuthorizer
+    val auth = if (isKRaftTest()) {
+      servers.last.authorizer.get
+    } else {
+      new AclAuthorizer
```

Comment:
Maybe we can use the authorizer of broker instead of creating new one?

Reply:
As I understand, this AclAuthorizer is used for some tests like `testAclCliWithAuthorizer`, `testProducerConsumerCliWithAuthorizer`, `testAclsOnPrefixedResourcesWithAuthorizer`, `testPatternTypesWithAuthorizer`. Those tests don't use AdminClient, so for them there seems to be no broker created at all, only Zookeeper instance. So there is no broker/controller from which we can obtain the AclAutorizer, hence creating a new instance of the autorizer here